resources/build/bootstrap-ubuntu-20.04.sh

```diff
@@ -7,19 +7,6 @@ sudo apt-get update
 # Install gcc, linters, build tools used by conan and Python 3.
```

Comment:
[Question] Is there anything in the "python 3" part of that comment that needs updating?

Reply:
I guess its still relevant since it's installing `pip`, which is needed on the next line. 

We could probably get rid of `build-essential pkgconf` now, since none of our Conan dependencies need building... and could get rid of the `--build=missing` from the `conan` command line too.

Reply:
Ah, though this file is still used as part of the Vagrant VM instructions, so `build-essential` is probably required for that.

Reply:
Cool beans, lets leave as is.